state-chain/pallets/cf-broadcast/src/tests.rs

```diff
@@ -1387,3 +1388,26 @@ fn should_release_barriers_correctly_in_case_of_rotation_tx_succeeding_first() {
 		assert_eq!(BroadcastBarriers::<Test, Instance1>::get(), BTreeSet::new());
 	});
 }
+
+#[test]
+fn changing_broadcast_timeout() {
+	new_test_ext().execute_with(|| {
+		// ensure that timeout is the default value
+		assert_eq!(crate::BroadcastTimeout::<Test, _>::get(), crate::mock::BROADCAST_EXPIRY_BLOCKS);
+
+		// update the timeout
+		const NEW_TIMEOUT: u32 = 100;
```

Comment:
nit: also good to assert:
```rust
assert_ne!(crate::mock::BROADCAST_EXPIRY_BLOCKS, NEW_TIMEOUT);
```
since the `BROADCAST_EXPIRY_BLOCKS` is not local to the test, to avoid accidental false positives in the future

Reply:
Yes, nice.